mitodl/ocw-hugo-themes
# added topics partial to course-v2-home to render topics in a hierarchy

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #812

#### What's this PR do?
This pr adds the `topics` partial to course v2 home page. Which causes the topics listed in the home page to be rendered in hierarchical order.

#### How should this be manually tested?
- Build any v2 course locally or view [netlify course v2](https://ocw-hugo-themes-course-v2-pr-818--ocw-next.netlify.app/) 
- Visit home page
- Verify topics are rendered hierarchical order

#### Any background context you want to provide?
In Course theme version 2, we moved away from rendering topics on the home page in a single line with `>` delimiter to using the same format we use to render course topics in course info. To achieve that result, we replace topics_online partial with topics partial. 

#### Screenshots (if appropriate)
<img width="1437" alt="Screenshot 2022-08-10 at 3 03 39 PM" src="https://user-images.githubusercontent.com/87968618/183884235-93fb88e2-cc1e-425b-bb18-d89d48efbe01.png">
<img width="1437" alt="Screenshot 2022-08-10 at 3 03 48 PM" src="https://user-images.githubusercontent.com/87968618/183884918-9b20bff5-8b91-464f-80ef-482fc403cdf9.png">
